app/views/questions/_answer.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="answers-container" style="margin-bottom:30px;">
+  <div id="a<%= answer.id %>" style="border: 1px solid #337ab7; border-radius:5px;">
```

Comment:
Hmm, is there a way we could break this out into a standard "comment header" or "response header" and not duplicate this with the comment header template?